starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,53 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
```

Comment:
te hizo falta llamar a tu archivo css
```suggestion
<head>
<link rel="stylesheet" href="./css/main.css">
```